db/postgresql/schema.sql

```diff
@@ -319,20 +333,55 @@ CREATE TABLE "sch_usr_notification_email"
 );
 CREATE INDEX "ix_SCH_USR_NOTIFICATION_EMAIL_email_id" ON "sch_usr_notification_email" ("email_id");
 
+CREATE SEQUENCE "seq_organization" START 1;
+CREATE TABLE "organization"
+(
+  "id" INTEGER,
+  "name" varchar(256) NOT NULL,
+  PRIMARY KEY ("id"),
+  UNIQUE ("name")
+);
+CREATE INDEX "ix_organization_name" ON "organization" ("name");
+
 CREATE SEQUENCE "seq_usr_user" START 1;
 CREATE TABLE "usr_user"
 (
   "id" INTEGER,
-  "username" VARCHAR(25) NOT NULL,
+  "username" VARCHAR(256) NOT NULL,
   "email_id" INTEGER,
-  "u_password" CHAR(142) NOT NULL,
-  "full_name" VARCHAR(55),
+  "u_password" CHAR(256) NOT NULL,
+  "full_name" VARCHAR(256),
   "table_rows" INTEGER,
   PRIMARY KEY ("id"),
   UNIQUE ("username")
 );
 CREATE INDEX "ix_USR_USER_email_id" ON "usr_user" ("email_id");
 
+CREATE SEQUENCE "seq_permission" START WITH 1;
+CREATE TABLE "permission" (
+  "id" INTEGER,
+  "name" varchar(142) NOT NULL,
+  "rwonly" boolean,
```

Comment:
What is the purpose of this column in the table of permission? For just one permission (pipeline.save) it is equal to 1, otherwise 0. So for me it seems to be rather redundant column - if we need to distinguish two states for "pipeline.save", why we do not introduce two roles for pipeline.save? Furthermore, such extra rendundant columns then causes rather magical "if" in DefaultPermissionEvaluator: "if (foundPermission.isRwOnly() && !ShareType.PUBLIC_RW.equals(sTarget.getShareType())) {", which is hard to understand